### PR TITLE
Updating rounting.py

### PR DESCRIPTION
**Pull request by gileadecastro**
Added ".as_wsgi()"

### CODE REVIEW

**Review by josnin — COMMENTED** [review]
Not clear on the intention :)

File: chat/routing.py [hunk 1]
```diff
@@ -3,5 +3,5 @@
 from . import consumers
 
 websocket_urlpatterns = [
-    re_path(r'ws/chat/(?P<room_name>\w+)/$', consumers.ChatConsumer),
+    re_path(r'ws/chat/(?P<room_name>\w+)/$', consumers.ChatConsumer.as_wsgi()),
```
[the remark this answers is not in the document]
Comment (suhailvs) [reply chat/routing.py:6]: django channels 3+ requires `.as_wsgi()`

[the remark this answers is not in the document]
Comment (josnin) [reply chat/routing.py:6]: this repo outdated still using django channels 2.4.0

may need to upgrade the repo to django channel 3

[the remark this answers is not in the document]
Comment (suhailvs) [reply chat/routing.py:6]: if you are merging this pullrequest, then you must also need to update `requirements` with `channels==2.4.0` -> `channels==3.0.0`

**Comment by josnin** [comment]
decided to combine all related upgrade (channel 2 to 3) into single commit

Thanks @suhailvs @gileadecastro ,

**Pull request closed without merging**